warehouse/templates/rss/project_updates.xml

```diff
@@ -0,0 +1,14 @@
+{% extends "base.xml" %}
+{% block title %}Recent Updates of {{ project_name }} package{% endblock %}
+{% block description %}Recent Updates of {{ project_name }} to the Python Package Index{% endblock %}
```

Comment:
It might be better if this title and description said something like "Latest releases of the `{{ project_name }}` package".

At the very least, this should say "updates **to the** `{{ project_name }}` package" instead of "updates **of** `{{ project_name }}` package"